### ISSUE TITLE
missing Collection `createIndex`?

### ISSUE THREAD

**Issue by GCSBOSS** [erfanium/atlas_sdk#30]
Is there any possibility top add the `createIndex` command for collection?

**Comment by erfanium** [comment]
@GCSBOSS Hey, thanks for reaching out

Do have any idea of how we can implement it? is there any api?

**Comment by objctp** [comment]
I haven't tested it myself, but you should be able to add support for the `createIndexes` operation in the `Collection` class by implementing it as follows:
```typescript
async createIndexes(indexes: Document[]): Promise<Document[]> {
  return this.callApi("createIndexes", { indexes });
}

**Comment by erfanium** [comment]
@juratori Can you send me the docs for `createIndexes` api route? I can't find anything related here https://www.mongodb.com/docs/atlas/api/data-api-resources/


It may works, but it's not documented anywhere

**Comment by objctp** [comment]
@erfanium you are right! `createIndexes` doesn't seem to be a built-in route within the '[Atlas Data API](https://www.mongodb.com/docs/atlas/api/data-api-resources/)'. Instead, I found that it is documented in the '[MongoDB Manual](https://www.mongodb.com/docs/manual/reference/command/createIndexes/#definition)' and is considered a part of the '[MongoDB Atlas Administration API](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Atlas-Search)'

It [appeared](https://github.com/denodrivers/mongo/blob/6e27e382448de56660f396a293d71efda6b6ce80/src/collection/collection.ts#L331) to me that the `createIndexes` operation is available for use, but it may not be as straightforward as I initially assumed. Implementing it could involve additional steps, such as dropping and listing indexes, and so on.
